src/game/area.c

```diff
@@ -410,6 +424,10 @@ void render_game(void) {
         } else
             gDPSetScissor(gDisplayListHead++, G_SC_NON_INTERLACE, 0, gBorderHeight, SCREEN_WIDTH,
                           SCREEN_HEIGHT - gBorderHeight);
```

Comment:
Unrelated, but now that I'm seeing it, please can this `else` match the `if` in terms of bracket usage (or lack thereof)?